From d0a510f0f5ee84cdc5fe4480cb7289d2b3486c60 Mon Sep 17 00:00:00 2001
From: Yu Yue <yuyue@mprc.pku.edu.cn>
Date: Mon, 31 Oct 2011 17:58:43 +0800
Subject: [PATCH 050/641] UniCore64: Add function time_init

Fixed error:
noinitramfs.c:(.init.text+0xb50): undefined reference to `time_init'

Signed-off-by: Yu Yue <yuyue@mprc.pku.edu.cn>
---
 arch/unicore64/kernel/Makefile |    1 +
 arch/unicore64/kernel/time.c   |   11 +++++++++++
 2 files changed, 12 insertions(+)
 create mode 100644 arch/unicore64/kernel/time.c

diff --git a/arch/unicore64/kernel/Makefile b/arch/unicore64/kernel/Makefile
index da7ca0c..e8a961f 100644
--- a/arch/unicore64/kernel/Makefile
+++ b/arch/unicore64/kernel/Makefile
@@ -1,3 +1,4 @@
 obj-y			:= ptrace.o setup.o init_task.o process.o
+obj-y			+= time.o
 head-y			:= head.o
 extra-y			:= $(head-y) vmlinux.lds
diff --git a/arch/unicore64/kernel/time.c b/arch/unicore64/kernel/time.c
new file mode 100644
index 0000000..d123e31
--- /dev/null
+++ b/arch/unicore64/kernel/time.c
@@ -0,0 +1,11 @@
+#include <linux/kernel.h>
+#include <linux/init.h>
+
+/**
+ * time_init() -
+ */
+void __init time_init(void)
+{
+	/* FIXME */
+	BUG();
+}
-- 
1.7.9.5

